x-pack/plugins/uptime/public/components/common/charts/ping_histogram.tsx

```diff
@@ -189,12 +193,22 @@ export const PingHistogramComponent: React.FC<PingHistogramComponentProps> = ({
 
   const pingHistogramExploratoryViewLink = createExploratoryViewUrl(
     {
-      'pings-over-time': {
-        dataType: 'synthetics',
-        reportType: 'kpi-over-time',
-        time: { from: dateRangeStart, to: dateRangeEnd },
-        ...(monitorId ? { filters: [{ field: 'monitor.id', values: [monitorId] }] } : {}),
-      },
+      reportType: 'kpi-over-time',
```

Comment:
This code runs but the link is never displayed, is it planned for a future iteration?

Reply:
yes, it's mostly for testing purpose for now, but yeah we will use this hopefully in 7.15 iteration